### PR TITLE
Plugin Generator Updates

### PR DESCRIPTION
**Pull request by Fryguy**
Plugin Generator Updates

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,17 +1,12 @@
-source 'https://rubygems.org'
-
+# Declare your gem's dependencies in manageiq-api.gemspec.
+# Bundler will treat runtime dependencies like base dependencies, and
+# development dependencies will be added by default to the :development group.
 gemspec
 
-# Load Gemfile with dependencies from manageiq
```
Comment (bdunne) [inline Gemfile:5]: Should this have changed?

Comment (Fryguy) [reply Gemfile:5]: Yes...the plugin generator no longer has this.  A lot of it is ceremony that is now handled by bin/setup and bin/update

Comment (Fryguy) [reply Gemfile:5]: Note we still load it (see line https://github.com/ManageIQ/manageiq-api/pull/886/files/390224f129d78f202a5da097a968d7ac42bcc0aa#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R11-R12 )

**Comment by Fryguy** [comment]
Note that the README changes are removed because they are all in the plugin developer doc.

**Review by agrare — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (agrare) [reply Gemfile:5]: I think this is a helpful error message, but we should move it to the generator and keep it consistent

**Pull request merged**